libs/runtime/ebpf_ring_buffer.c

```diff
@@ -178,6 +192,11 @@ ebpf_ring_buffer_return(_Inout_ ebpf_ring_buffer_t* ring, size_t length)
     }
     // Did it end on a record boundary?
     if (local_length != 0) {
+        EBPF_LOG_MESSAGE_UINT64(
+            EBPF_TRACELOG_LEVEL_ERROR,
+            EBPF_TRACELOG_KEYWORD_MAP,
+            "ebpf_ring_buffer_return: Invalid length",
```

Comment:
Unless I am misreading here, it looks like this is the same trace statement as line 177. Do we want to trace a (slightly) different message so that we can identify exactly which failure was hit?

Reply:
On second read - I see we have 2 arguments in the above compared to just 1 here. Still, I'd probably suggest clearer trace statements to help improve debuggability.

Reply:
Fixed.